Jenkinsfile.upstream-operator-sync

```diff
@@ -0,0 +1,166 @@
+@Library('jenkins-pipeline-shared-libraries')_
+
+boolean goModModified = false
+operatorTagVersion = "upstream-sync"
+
+pipeline {
+    agent {
+        label 'kogito-operator-slave && !master'
+    }
+    // Needed for local build
+    tools {
+        jdk 'kie-jdk11'
+    }
+    options {
+        timeout(time: 90, unit: 'MINUTES')
+    }
+    environment {
+        KOGITO_CI_EMAIL_TO = credentials("${JENKINS_EMAIL_CREDS_ID}")
+        OPENSHIFT_INTERNAL_REGISTRY = 'image-registry.openshift-image-registry.svc:5000'
+
+        CONTAINER_TLS_OPTIONS="--tls-verify=false"
```

Comment:
defined also here: https://github.com/kiegroup/rhpam-kogito-operator/pull/22/files#diff-954fb9904df55eadd8693b9166aadf6ebb64b995f8c366ed1a1b7be597aeacb3R78
I guess you can move to job definition